### ISSUE TITLE
Unable to use NotoColorEmoji: invalid pixel size

### ISSUE THREAD

**Issue by jdhao** [python-pillow/Pillow#6166]
### What did you do?

I am trying to use NotoColorEmoji font from [here](https://github.com/googlefonts/noto-emoji/raw/main/fonts/NotoColorEmoji.ttf).
Here is my code:

```
from PIL import ImageFont

FONT_PATH = './NotoColorEmoji.ttf'
font_size = 20
font = ImageFont.truetype(FONT_PATH, size=font_size)
```

### What did you expect to happen?

I can open the font without error.

### What actually happened?

I am seeing the following error:

```
     font = ImageFont.truetype(FONT_PATH, size=font_size)
   File "/home/jdhao/tools/miniconda/lib/python3.7/site-packages/PIL/ImageFont.py", line 844, in truetype
     return freetype(font)
   File "/home/jdhao/tools/miniconda/lib/python3.7/site-packages/PIL/ImageFont.py", line 841, in freetype
     return FreeTypeFont(font, size, index, encoding, layout_engine)
   File "/home/jdhao/tools/miniconda/lib/python3.7/site-packages/PIL/ImageFont.py", line 194, in __init__
     font, size, index, encoding, layout_engine=layout_engine
 OSError: invalid pixel size
```

### What are your OS, Python and Pillow versions?

* OS: CentOS 7.4
* Python: 3.7.6
* Pillow: 9.0.0

```python
code goes here
```

**Comment by nulano** [comment]
https://github.com/python-pillow/Pillow/issues/3346#issuecomment-612139601

> This error is due to a freetype limitation. Bitmap fonts don't support scaling with freetype, so you must specify a valid size, which is 109 for Noto Color Emoji.

**Comment by nulano** [comment]
Google has proposed a new scalable format for colored fonts, COLR v1.

An experimental version of Noto Color Emoji should be available here: https://github.com/googlefonts/color-fonts
However, I don't think FreeType has full support for COLR v1 yet.

**Comment by radarhere** [comment]
> However, I don't think FreeType has full support for COLR v1 yet.

No, from what I can see, "an experimental interface" was added in 2.11.0.

@jdhao you liked the previous comment. Has this answered your question?

**Comment by jdhao** [comment]
@radarhere The ultimate goal is to use the color emoji font as expected. Juding from the comment, this is not possible for now? My upvote is just for the clarification 😂

**Comment by jdhao** [comment]
Okay, based on code [this pr](https://github.com/python-pillow/Pillow/pull/4955), I figured out. We need to change font size to exactly 109, then it should be fine.

```
from PIL import ImageFont

FONT_PATH = './NotoColorEmoji.ttf'
font_size = 109
font = ImageFont.truetype(FONT_PATH, size=font_size)
```
